src/apps/filters/h3.c

```diff
@@ -102,6 +107,7 @@ bool latLngToCellCmd(int argc, char *argv[]) {
     H3Index c;
     H3Error e = H3_EXPORT(latLngToCell)(&ll, res, &c);
 
+    // TODO: Add support for JSON formatting
```

Comment:
Does JSON formatting mean `"8928342e20fffff"` in this case? Also, does `h3Println` make piping harder than `h3Print`?

Reply:
That or `{"cell": "8928342e20fffff"}`. I'm not sure which would be a better idea.